x-pack/plugins/fleet/public/services/get_template_url_from_agent_policy.ts

```diff
@@ -7,21 +7,19 @@
 
 import type { AgentPolicy } from '../types';
 
-/**
- * Get the cloud formation template url from a agent policy
- * It looks for a config with a cloud_formation_template_url object present in
- * the enabled package_policies inputs of the agent policy
- */
-export const getCloudFormationTemplateUrlFromAgentPolicy = (selectedPolicy?: AgentPolicy) => {
+export const getTemplateUrlFromAgentPolicy = (
+  templateUrlFieldName: string,
```

Comment:
we can also export the supported `templateUrlFieldName` in this function similar as mentioned in the previous comment